src/git.js

```diff
@@ -1,37 +1,57 @@
 const { exec } = require('child_process');
+const path = require('path');
 const util = require('util');
+const utils = require('./utils');
 const logger = require('./logger');
 const { spawnProcess } = require('../processHelper');
 
 const bashExec = util.promisify(exec);
 
-const getFiles = async (path) => {
-  const { stdout, stderr } = await bashExec(`git ls-files ${path} ':!:${path}/basket-of-unused-content*'`);
-
-  if (stderr) return logger.info({ message: `Error at getFiles because: ${stderr}` });
-
-  return stdout;
+const getGitRoot = async (dirOrFile) => {
+  try {
+    const directory = path.dirname(dirOrFile);
+    const { stdout } = await bashExec('git rev-parse --show-toplevel', { cwd: directory });
+    return stdout.trim();
+  } catch (error) {
+    return logger.info({ message: `Error at getGitRoot: ${error}` });
+  }
 };
 
-const getCommitId = async (path) => {
-  const { stdout, stderr } = await bashExec(`git log -n 1 --pretty=format:%H -- ${path}`);
-
-  if (stderr) return logger.info({ message: `Error at getCommitIds because: ${stderr}` });
-
-  return stdout;
+const getFiles = async (directory) => {
+  try {
+    const gitRoot = await getGitRoot(directory);
+    const { stdout } = await bashExec('git ls-files \':!:./basket-of-unused-content*\' --full-name', { cwd: directory });
+    const sanitizedArrayOfFiles = utils.sanitizeFilesArray(stdout);
+    return sanitizedArrayOfFiles.map((f) => path.join(gitRoot, f));
+  } catch (error) {
+    return logger.info({ message: `Error at getFiles because: ${error.stderr}` });
```

Comment:
Sugestão so pra padronizar os logs
```suggestion
    return logger. error({ message: `Error at getFiles because: ${error.stderr}` });
```